app/models/iam_user.rb

```diff
@@ -78,7 +78,7 @@ def set_expire
   def vet360_contact_info
     super
   rescue Faraday::ResourceNotFound
-    raise Common::Exceptions::RecordNotFound.new(id: vet360_id)
+    raise Common::Exceptions::RecordNotFound.new({ id: vet360_id })
```

Comment:
```suggestion
    raise Common::Exceptions::RecordNotFound.new(vet360_id)
```

Looking at `Common::Exceptions::RecordNotFound`, I think this should be ^